### PR TITLE
CORTX-33279  F-40E 2 Tests TA

### PR DESCRIPTION
**Pull request by RajeshPratapSInghChouhan**
# Problem Statement
[CORTX-33279](https://jts.seagate.com/browse/CORTX-33279) TA : F-40E CSM : Deployment: Config: Resource Limits Part-1

# Design
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/1077084998/QA+F-40E+CSM+Resource+Limits+tests

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [x] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [x] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by kachhwahadivya — APPROVED** [review]

File: tests/csm/rest/test_csm_load.py [hunk 1]
```diff
@@ -69,6 +70,8 @@ def setup_class(cls):
         assert s3acc_already_present
         cls.log.info("[Completed]: Setup class")
         cls.default_cpu_usage = False
+        cls.iam_users_created = []
+        cls.request_usage = 122
```
Comment (RajeshPratapSInghChouhan) [inline tests/csm/rest/test_csm_load.py:75]: @kachhwahadivya will add a function to get this from deployment

File: tests/csm/rest/test_csm_load.py [hunk 2]
```diff
@@ -277,6 +293,130 @@ def test_44257(self):
...
+        self.log.info("New users to create: %s", new_iam_users)
+        thread_request = test_cfg["thread_request"] # ( login + create )
+        operation_count = new_iam_users * thread_request
+
+        request_usage_rounded = int( thread_request * round( self.request_usage / thread_request ))
+        loop = operation_count // request_usage_rounded
+        req_in_loops = request_usage_rounded * loop
+        operation_last = operation_count - req_in_loops
+        threads_1 = request_usage_rounded // thread_request
+        threads_2 = operation_last // thread_request
```
Comment (RajeshPratapSInghChouhan) [inline tests/csm/rest/test_csm_load.py:330]: @kachhwahadivya have a lib function to do this less complicated and clean, will be updated later

**Review by sonalk0209 — APPROVED** [review]
Approving for merge

**Comment by RajeshPratapSInghChouhan** [comment]
pytest --local True --collect-only
:::::::::::::::::
] created at /root/jenkins/workspace/QA/QA_R2_Single_Runner_CSM_error_msg_check/cortx-test/log/te_meta.json
Successfully unmounted directory

================================================================ 2361 tests collected in 13.86s =================================================================
gYou have new mail in /var/spool/mail/root
(virenv) [root@ssc-vm-g2-rhev4-1909 cortx-test]# git branch 
  CORTX-33276_TA_F-40E_3_Tests
* CORTX-33279_TA_F-40E_Tests

**Pull request merged**